src/components/tracked-controls.js

```diff
@@ -169,8 +169,13 @@ module.exports.Component = registerComponent('tracked-controls', {
       object3D.matrixAutoUpdate = false;
       object3D.matrix.compose(object3D.position, object3D.quaternion, object3D.scale);
       object3D.matrix.multiplyMatrices(standingMatrix, object3D.matrix);
-      object3D.matrixWorldNeedsUpdate = true;
+      object3D.matrix.decompose(object3D.position, object3D.quaternion, object3D.scale);
     }
+
+    object3D.rotateZ(this.data.rotationOffset * THREE.Math.DEG2RAD);
+
+    object3D.updateMatrix();
```

Comment:
I haven't tested this PR in a Daydream/GearVR with a 3DoF controller yet. We disable `matrixAutoUpdate` in `init`, so I'm not sure how this code could have worked for those controllers without an explicit `updateMatrix`. Perhaps it _was_ actually broken. I'll test with a 3DoF controller when I get a chance.

Reply:
Confirmed. Daydream controller does not update without this.

Reply:
Thanks for testing on it.